pkg/kncloudevents/v2/cloudevents_test.go

```diff
@@ -14,31 +14,22 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package adapter
```

Comment:
you are not suppose to use v2 as the package, it should be kncloudevents

Reply:
fixed.